include/swift/AST/EducationalNotes.def

```diff
@@ -91,5 +91,8 @@ EDUCATIONAL_NOTES(result_builder_missing_build_either,
                   "result-builder-methods.md")
 EDUCATIONAL_NOTES(result_builder_missing_build_array,
                   "result-builder-methods.md")
-
+                  
+EDUCATIONAL_NOTES(multiple_inheritance,
+                  "multiple-class-inheritance.md")
```

Comment:
```suggestion
                  "multiple-inheritance.md")
```

This is just a suggestion, but I think the term of art ("multiple inheritance") is sufficiently established that it's better to keep it together without elaboration; that makes it searchable, etc.